store/aciinfo.go

```diff
@@ -21,6 +21,16 @@ import (
 	"time"
 )
 
+// ACIFetchInfo is used to store information about the fetching process of an ACI.
+type ACIFetchInfo struct {
+	// Latest defines if the ACI was imported using the latest pattern
+	// (no version label was provided on ACI discovery).
+	Latest bool
+
+	// InsecureOptions are the insecure options that was used when retrieving the ACI.
+	InsecureOptions int64
+}
```

Comment:
This is creating a relation/dependency between the store value with the values defined in `rkt/flag/secflags.go`. I'm not sure this dependency is a good thing (also for store reuse with other projects) and this means that we must be sure that the flags won't change in the future (only add) or they will cause problems and/or report wrong flags. If this can be assured than probably a comment in `rkt/flag/secflags.go` should be added.

In the new store I'll rework this to use some "store user" (rkt, acbuild) custom fields (since it'll probably use a json document instead of a strict schema).